### DISCUSSION TITLE
where is the old docs website?

### DISCUSSION THREAD

**Discussion by AntonyOnScript** [unocss/unocss#2050] [Q&A] [answered]
i think the old docs website was really good to find what i was looking for, unlike the current one.

**Comment by zyyv** [comment, marked as answer]
Since the advent of interactive docs, we have abandoned the traditional documentation site. We are compatible with `tailwind` and `windi`, you can check their docs if you need.😅

**Reply by jojojojojoj5564656465465** [reply]
It would be great a doc on the config file
